core/interfaces.go

```diff
@@ -91,6 +91,10 @@ type Services interface {
 	GetResearchProfileUserCount(clientID string, current *model.User, researchProfile map[string]map[string][]string) (int64, error)
 }
 
+// BBs exposes Building Block APIs for the driver adapters
```

Comment:
For the moment we can remove this since we are not using it yet. At some point we can come back and restructure these interfaces to be more like the Core BB